contract/src/Custody.sol

```diff
@@ -404,6 +429,9 @@ contract Custody is IChannel, IDeposit {
         if (meta.stage == Status.VOID) revert ChannelNotFound(channelId);
```

Comment:
Add `proof` (either array or a single state) parameter to `resize(...)` to give more security guarantees.